frontend/views/components/GroupSettings/InviteActionButton.vue

```diff
@@ -0,0 +1,60 @@
+<template lang="pug">
+menu-parent
+  menu-trigger.is-icon
+    i.icon-ellipsis-v
+
+  menu-content.c-content
+    ul
+      menu-item(
+        tag='router-link'
+        item-id='hash-3'
+        to='/somewhere'
+      )
+        i18n.icon-vote-yea.c-icon(tag='i') See original proposal
+      menu-item(
+        v-if='!linkExpired'
+        tag='router-link'
+        item-id='hash-3'
+        to='/somewhere'
+      )
+        i18n.icon-times.c-icon(tag='i') Revoke Link
```

Comment:
I wouldn't mind having this component directly as part of InviteLinks. It's not very complex and won't be reused anywhere else... but it's up to you :)

Reply:
That makes sense. I deleted the action button component.